studio/components/interfaces/BillingV2/Subscription/Tier/Tier.constants.ts

```diff
@@ -17,7 +17,8 @@ export const SUBSCRIPTION_PLANS: PricingInformation[] = [
     preface: undefined,
     features: [
       'Social OAuth providers',
-      'Up to 500MB database & 1GB file storage',
+      'Up to 500MB database space',
```

Comment:
This would introduce inconsistencies with our pricing page or other places - if we wanna adjust this, then we'd have to do it everywhere

Reply:
These pricing bullet points are mainly on Rors, so best doublecheck with him if you wanna separate these two